scripts/ios/carthage.js

```diff
@@ -0,0 +1,38 @@
+module.exports = function(ctx) {
+    if (ctx.opts.cordova.platforms.indexOf('ios') == -1) {
```

Comment:
With JS, it's better to use "===" instead of "=="
https://www.impressivewebs.com/why-use-triple-equals-javascipt/